tssc/step_implementers/deploy/argocd.py

```diff
@@ -321,11 +320,29 @@ def _run_step(self):
                 argocd_app_name,
                 _out=sys.stdout)
 
+            # NOTE: Creating a file to pass to the next step
+            manifest_file = self.write_working_file(
+                'deploy_argocd_manifests.yml',
+                b''
+            )
+
+            sh.argocd.app.manifests(argocd_app_name, _out=manifest_file) # pylint: disable=no-member
+
             results = {
-                'argocd-app-name': argocd_app_name,
-                'config-repo-git-tag' : self._get_tag(repo_directory),
-                'argocd-endpoint-url': 'http://{endpoint}'.format(
-                    endpoint=self._get_endpoint_url())
+                'result': {
+                    'success': True,
+                    'message': 'deploy step completed - see report-artifacts',
+                    'argocd-app-name': argocd_app_name,
+                    'argocd-endpoint-url': 'http://{endpoint}'.format(
```

Comment:
Apologies for not catching this in the review of [NAPSSPO-954](https://github.com/rhtconsulting/tssc-python-package/pull/71), but [this file's doc](https://github.com/rhtconsulting/tssc-python-package/blob/569e1e83320a758e7566c05271a857267463c3fd/tssc/step_implementers/deploy/argocd.py#L78) needs to be updated to match the addition of `argocd-endpoint-url` to the results output.

Reply:
updated